package.json

```diff
@@ -26,19 +33,28 @@
   },
   "homepage": "https://github.com/eslint/eslint.github.io#readme",
   "dependencies": {
-    "moment": "^2.24.0"
+    "moment": "^2.24.0",
```

Comment:
I don't think this is actually used anywhere...Will remove.

Reply:
This was used in the scripts that were pulling sponsor info, I think now that we've switched to GraphQL API, it's no longer needed.